engine/installation/linux/ubuntulinux.md

```diff
@@ -292,7 +292,9 @@ To create the `docker` group and add your user:
 		Cannot connect to the Docker daemon. Is 'docker daemon' running on this host?
 
 	Check that the `DOCKER_HOST` environment variable is not set for your shell.
-	If it is, unset it.
+	If it is, unset it. Also make sure that you aren't keeping ssh socket files with a timeout. 
+	You can wait for the sockets files to be removed as per specified timeout or manually delete
```

Comment:
Can you include a little more info about how to make sure that you aren't keeping the ssh sockert files around? Like what do you need to look for in your SSH config or on the filesystem? 

Also consider rephrasing to something like "Verify that SSH socket files are not retained due to timeout settings."